src/Bicep.LangServer/Completions/BicepCompletionContextKind.cs

```diff
@@ -138,5 +138,10 @@ public enum BicepCompletionContextKind
         /// We're at this place in an import statement: 'import foo from |'
         /// </summary>
         ImportFromFollower = 1 << 24,
+
+        /// <summary>
+        /// We're inside dependsOn array
+        /// </summary>
+        ResourceOrModuleItem = 1 << 25,
```

Comment:
>ResourceOrModuleItem [](http://example.com/codeflow?start=8&length=20)

We should probably make the name more specific. How about `DependsOn` or `SymbolicName`?

Reply:
I like SymbolicName. Thanks!